docs/configuration.md

```diff
@@ -70,11 +70,12 @@ $configuration = Configuration::forSymmetricSigner(
 
 Currently supported symmetric algorithms:
 
-| Name      | Description        | Class                                    | Key length req. |
-| --------- | ------------------ | ---------------------------------------- | --------------- |
-| `HS256`   | HMAC using SHA-256 | `\Lcobucci\JWT\Signer\Hmac\Sha256`       | 32 bytes        |
-| `HS384`   | HMAC using SHA-384 | `\Lcobucci\JWT\Signer\Hmac\Sha384`       | 48 bytes        |
-| `HS512`   | HMAC using SHA-512 | `\Lcobucci\JWT\Signer\Hmac\Sha512`       | 64 bytes        |
+| Name      | Description        | Class                                | Key length req. | Note                                                                                                       |
+| --------- | ------------------ | ------------------------------------ |-----------------|------------------------------------------------------------------------------------------------------------|
+| `HS256`   | HMAC using SHA-256 | `\Lcobucci\JWT\Signer\Hmac\Sha256`   | 32 bytes        |                                                                                                            |
+| `HS384`   | HMAC using SHA-384 | `\Lcobucci\JWT\Signer\Hmac\Sha384`   | 48 bytes        |                                                                                                            |
+| `HS512`   | HMAC using SHA-512 | `\Lcobucci\JWT\Signer\Hmac\Sha512`   | 64 bytes        |                                                                                                            |
+| `BLAKE2B` | Blake2b keyed Hash | `\Lcobucci\JWT\Signer\Hmac\Blake2b`  | 16 bytes        | Not a [JWT standard](https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms) |
```

Comment:
I added the min length required by libsodium... should we align with HMAC algos and bump this to 32?

Reply:
Sounds reasonable

Reply:
Handled :+1: